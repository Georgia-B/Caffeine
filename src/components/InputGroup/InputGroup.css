.input__group {
    display: flex;
    align-items: center;
    justify-content: center;
    padding: 15px 0;
    flex-wrap: wrap;
}