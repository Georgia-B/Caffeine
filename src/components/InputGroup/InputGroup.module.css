.inputgroup {
    display: flex;
    align-items: center;
    justify-content: center;
    padding: 15px 0;
    flex-wrap: wrap;
}