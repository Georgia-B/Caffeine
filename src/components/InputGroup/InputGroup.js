import React from 'react';
import PropTypes from 'prop-types';
import './InputGroup.css';

const InputGroup = ({ children }) => {
    return (
        <div className="input__group">
            {children}
        </div>
    );
};

InputGroup.propTypes = {
    children: PropTypes.node,
};

export default InputGroup;
