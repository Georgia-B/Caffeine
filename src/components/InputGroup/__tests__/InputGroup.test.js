import React from 'react';
import { cleanup, render } from '@testing-library/react';
import InputGroup from '../InputGroup';

describe('Components | <InputGroup />', () => {
    afterEach(cleanup);
    it('should render correctly', () => {
        const component = render(<InputGroup><div>Hello World</div></InputGroup>);

        expect(component.asFragment()).toMatchSnapshot();
    });
});
