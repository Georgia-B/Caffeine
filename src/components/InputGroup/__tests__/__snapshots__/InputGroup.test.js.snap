// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components | <InputGroup /> should render correctly 1`] = `
<DocumentFragment>
  <div
    class="inputgroup"
  >
    <div>
      Hello World
    </div>
  </div>
</DocumentFragment>
`;
